bundle/Resources/config/services/templating.yml

```diff
@@ -121,5 +121,7 @@ services:
         # public: false
         arguments:
             - '@netgen.ezplatform_site.named_object_provider'
+            - '@logger'
```

Comment:
Should be `@?logger`, since logger can be `null` if someone configures it so.

Reply:
Updated: https://github.com/netgen/ezplatform-site-api/pull/164/commits/0a7d31ee8fd53538aecd9263d0a2ce90f7da14e8